### PR TITLE
Bring README.md up to date. Closes #357

### LINKED ISSUE #357: Update Quick Start Guide - VirtualBox is installed by Vagrant
Installation is much simpler as of Vagrant 1.8.0—it will install VirtualBox automatically if it's not already present, so the Quick Start Guide can literally be updated to:

  1. Install Vagrant.
  2. Download or clone this project.
  3. `vagrant up`

Especially considering #355 (using the new `ansible_local` provisioner), Vagrant 1.8.0 has made things oh-so-much more amazing :)

### PR DESCRIPTION
**Pull request by oxyc**
**Not ready to be merged** yet as we should wait for Vagrant 1.8.2 so that `ansible_local` works reliably.

I mentioned that it's tested under 1.8.2 as of February 2016, so let's hope it gets released by then :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,4 +122,4 @@ This project is licensed under the MIT open source license.
 
 ## About the Author
 
-[Jeff Geerling](http://jeffgeerling.com/), owner of [Midwestern Mac, LLC](http://www.midwesternmac.com/), created this project in 2014 so he could accelerate his Drupal core and contrib development workflow. This project, and others like it, are also featured as examples in Jeff's book, [Ansible for DevOps](http://www.ansiblefordevops.com/).
+[Jeff Geerling](http://www.jeffgeerling.com/), owner of [Midwestern Mac, LLC](http://www.midwesternmac.com/), created this project in 2014 so he could accelerate his Drupal core and contrib development workflow. This project, and others like it, are also featured as examples in Jeff's book, [Ansible for DevOps](http://www.ansiblefordevops.com/).
```
Comment (geerlingguy) [inline README.md]: It's the little things :)

**Comment by geerlingguy** [comment]
Halfway through February... fingers crossed for a Vagrant 1.8.2 release... someday!

**Comment by oxyc** [comment]
I'm getting worried https://github.com/mitchellh/vagrant/issues/6726 wont get in before....

**Comment by geerlingguy** [comment]
@oxyc - At least that issue isn't breaking anything with Drupal VM in particular. It's definitely annoying, but it's not breaking my stuff :)

**Comment by oxyc** [comment]
But it's postponing this awesomeness https://github.com/geerlingguy/drupal-vm/pull/378!

**Comment by frob** [comment]
Does this also add travis ci test tags, now that #412 is merged?

**Comment by oxyc** [comment]
What are travis ci test tags?

**Comment by frob** [comment]
sorry, I should have been more specific. They are the build status tags that get added to the README, and I see them now. 

html tags
not git tags.

**Comment by geerlingguy** [comment]
I jumped the gun, merged this, and un-updated the Vagrant release to 1.8.1. In #445 I switched the provisioner away from ansible_local. I'll switch it back in #450.

**Pull request merged**